sdk/sdk-package/antpkg/build_batch_script.xml

```diff
@@ -0,0 +1,32 @@
+<project default="shell">
+  <target name="shell">
+    <echo>Building start batch script for [${id}]</echo>
+    <loadfile property="license" srcFile="license-header.txt">
+      <filterchain>
+        <prefixlines prefix="@REM"/>
+      </filterchain>
+    </loadfile>
+    <property name="fulloutput" value="${outputDir}/${binDir}"/>
+    <property name="binbaserelpath" basedir="${fulloutput}" value="${outputDir}" relative="true"/>
+    <copy file="antpkg/windowsBinTemplate" tofile="newShellScript"/>
+    <replace file="newShellScript" token="#LICENSE_HEADER#" value="${license}"/>
+    <replace file="newShellScript" token="#BIN_BASE_RELATIVE_PATH#" value="${binbaserelpath}"/>
+    <replace file="newShellScript" token="#ENV_SETUP#" value=""/>
+    <replace file="newShellScript" token="#REPO#" value="${repoName}"/>
+    <replace file="newShellScript" token="#JAVA_BINARY#" value="java"/>
+    <!-- TODO: Make the value of CLASSPATH variable according to appassembler plugin? -->
```

Comment:
appassembler has ton of functionality we don't necessarily need - if there is no obvious use case for populating all these variables, I would not bother

Reply:
Okay, then I will remove the TODO and if we ever need additional functionality we just open an issue.

Reply:
Yap. Also feel free to nuke appassembler from this repo. I believe I have an up to date fork with my patches there: https://github.com/dmarszk/appassembler